<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvalMainWindow</class>
 <widget class="QMainWindow" name="EvalMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EvalMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoad_data_table"/>
    <addaction name="actionSave_data_table"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <addaction name="actionAdjust_plot_appearance"/>
    <addaction name="actionSet_number_format"/>
    <addaction name="actionSet_path_to_EDII_service"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheck_for_update"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>E&amp;xport</string>
    </property>
    <addaction name="actionExport_plot_as_image"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionLoad_ChemStation_file">
   <property name="text">
    <string>&amp;Load ChemStation data</string>
   </property>
  </action>
  <action name="actionLoad_data_table">
   <property name="text">
    <string>Load &amp;data table</string>
   </property>
  </action>
  <action name="actionSave_data_table">
   <property name="text">
    <string>&amp;Save data table</string>
   </property>
  </action>
  <action name="actionSet_number_format">
   <property name="text">
    <string>&amp;Customize locale settings</string>
   </property>
  </action>
  <action name="actionAdjust_plot_appearance">
   <property name="text">
    <string>&amp;Adjust plot appearance</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExport_plot_as_image">
   <property name="text">
    <string>&amp;Export plot as image</string>
   </property>
  </action>
  <action name="actionExport_plot_data_as_CSV">
   <property name="text">
    <string>Export &amp;plot data as CSV</string>
   </property>
  </action>
  <action name="actionCheck_for_update">
   <property name="text">
    <string>&amp;Check for update</string>
   </property>
  </action>
  <action name="actionCsv_from_file">
   <property name="text">
    <string>&amp;From file</string>
   </property>
  </action>
  <action name="actionCsv_from_clipboard">
   <property name="text">
    <string>From &amp;clipboard</string>
   </property>
  </action>
  <action name="actionLoad_NetCDF_file">
   <property name="text">
    <string>Load &amp;NetCDF file</string>
   </property>
  </action>
  <action name="actionSet_path_to_EDII_service">
   <property name="text">
    <string>Set path to EDII service</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
